hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/CommonConfigurationKeysPublic.java

```diff
@@ -171,6 +171,13 @@
   public static final String  FS_TRASH_INTERVAL_KEY = "fs.trash.interval";
   /** Default value for FS_TRASH_INTERVAL_KEY */
   public static final long    FS_TRASH_INTERVAL_DEFAULT = 0;
+
+  public static final String MOVE_TO_TRASH_FOR_TEST_KEY = "move.to.trash.for.tests";
```

Comment:
would it be possible to not add a test only configuration key? Like use mockito to help the test?

Reply:
@jojochuang  MOVE_TO_TRASH_FOR_TEST_KEY，
This configuration is used as a switch. The default is off. It is only turned on when the unit is tested. It is the logic that is executed in the unit test. Other people write the same.What better advice do you have?Thank you.